components/header/navbar/SideMenu.tsx

```diff
@@ -8,14 +9,14 @@ type SideMenuProps = {
 const SideMenu = ({ open, children }: SideMenuProps) => {
   // TODO: Use framer
   return (
-    <div
-      className="fixed left-0 top-0 z-10 min-h-screen w-[90vw] overflow-x-hidden bg-darkgray px-8 py-16 duration-[0.4s] descendant:mb-2"
+    <motion.div
+      className="fixed left-0 top-0 z-10 min-h-screen w-full overflow-x-hidden bg-darkgray px-8 py-16 duration-[0.4s] descendant:mb-2 sm:w-[450px]"
```

Comment:
This would fix the bug where the background stays dark gray when the screen is made larger while the sidebar is open
```suggestion
      className="fixed left-0 top-0 z-10 min-h-screen w-full overflow-x-hidden bg-darkgray px-8 py-16 duration-[0.4s] descendant:mb-2 sm:w-[450px] lg:bg-transparent"
```